Mathlib/LinearAlgebra/DFinsupp.lean

```diff
@@ -310,6 +302,36 @@ end AddCommGroup
 
 end mapRange
 
+section domLCongr
+
+/-- `DFinsupp.equivCongrLeft` as a linear equivalence.
+
+This is the `DFinsupp` version of `Finsupp.domLCongr`. -/
+@[simps! apply]
```

Comment:
why only `simps! apply` when you use the declaration directly in a simp set below? What's bad about the `symm_apply` lemma?